packages/helm-charts/oracle/templates/oracle.cronjob.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: oracle
+    chart: oracle
+    release: {{ .Release.Service }}
+    component: oracle
+spec:
+  schedule: "{{ .Values.oracle.cronSchedule }}"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      template:
+        spec:
+          initContainers:
+          - name: get-current-price
+            image: {{ .Values.oracle.image.repository }}:{{ .Values.oracle.image.tag }}
+            imagePullPolicy: IfNotPresent
+            command:
+              - sh
+              - "-c"
+              - |
+                echo `./current_rate.sh` > /celo/.celo/current_price
```

Comment:
nit: should be able to just run `./current_rate.sh > /celo/.celo/current_price` right?